src/main/resources/db_scripts/create_analysis_transforms.sql

```diff
@@ -202,4 +202,14 @@ CREATE TABLE anonymous.teacher_engagement_v2_events AS
             OR event_type = 'VIEW_HINT' AND event_details ->> 'hintIndex' = '0'
         )
     UNION
-        (SELECT user_id, timestamp from anonymous.workshop_join_events);
\ No newline at end of file
+        (SELECT user_id, timestamp from anonymous.workshop_join_events);
+
+CREATE TABLE anonymous.england_scotland_state_schools_v2 AS
```

Comment:
This is fine to create a new table, but it won't inherit the indices of its base table. Whilst we don't want the indices created unnecessarily during the initial initialisation of the anon database (before sending), we may want to add an index and primary key to this table elsewhere?